<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisionerLab</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImLab">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_2"/>
   </widget>
   <widget class="QMenu" name="menuL">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionr"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuColor_Space">
     <property name="title">
      <string>Color Space</string>
     </property>
     <addaction name="actionBinary"/>
     <addaction name="actionGray"/>
     <addaction name="actionMap"/>
     <addaction name="actionRGB"/>
    </widget>
    <addaction name="actionInformation"/>
    <addaction name="actionAttributes"/>
    <addaction name="actionPalette"/>
    <addaction name="actionRename"/>
    <addaction name="menuColor_Space"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QMenu" name="menuThreshold">
     <property name="title">
      <string>Threshold</string>
     </property>
     <addaction name="actionManual"/>
     <addaction name="actionOtus"/>
     <addaction name="actionAdaptive"/>
    </widget>
    <widget class="QMenu" name="menuHistogram">
     <property name="title">
      <string>Histogram</string>
     </property>
     <addaction name="actionEqualization"/>
     <addaction name="actionExpansion"/>
    </widget>
    <widget class="QMenu" name="menuFilter">
     <property name="title">
      <string>Filter</string>
     </property>
     <widget class="QMenu" name="menuConvolve_Kernel">
      <property name="title">
       <string>Convolve Kernel</string>
      </property>
      <addaction name="actionSobel"/>
      <addaction name="actionScharr_3_3"/>
      <addaction name="actionLaplacian_5_5"/>
      <addaction name="actionLaplacian_7_7"/>
      <addaction name="actionGaussian_3_3"/>
      <addaction name="actionGaussian_5_5"/>
      <addaction name="actionGaussian_7_7"/>
      <addaction name="actionMean_3_3"/>
      <addaction name="actionMean_5_5"/>
      <addaction name="actionMean_7_7"/>
      <addaction name="actionMedian_3_3"/>
      <addaction name="actionMedian_5_5"/>
      <addaction name="actionMedian_7_7"/>
     </widget>
     <addaction name="menuConvolve_Kernel"/>
    </widget>
    <addaction name="actionSize"/>
    <addaction name="actionGeometric"/>
    <addaction name="actionQuantize"/>
    <addaction name="actionColor_Compon"/>
    <addaction name="menuThreshold"/>
    <addaction name="menuHistogram"/>
    <addaction name="menuFilter"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuImLab"/>
   <addaction name="menuL"/>
   <addaction name="menuView"/>
   <addaction name="menuImage"/>
   <addaction name="menuProcess"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionr">
   <property name="text">
    <string>r</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionAttributes">
   <property name="text">
    <string>Attributes</string>
   </property>
  </action>
  <action name="actionPalette">
   <property name="text">
    <string>Palette</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="text">
    <string>Binary</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionMap">
   <property name="text">
    <string>Map</string>
   </property>
  </action>
  <action name="actionRGB">
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionsave_2">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="text">
    <string>Size</string>
   </property>
  </action>
  <action name="actionGeometric">
   <property name="text">
    <string>Geometric</string>
   </property>
  </action>
  <action name="actionQuantize">
   <property name="text">
    <string>Quantize</string>
   </property>
  </action>
  <action name="actionColor_Compon">
   <property name="text">
    <string>Color Components</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionOtus">
   <property name="text">
    <string>Otus</string>
   </property>
  </action>
  <action name="actionAdaptive">
   <property name="text">
    <string>Adaptive</string>
   </property>
  </action>
  <action name="actionEqualization">
   <property name="text">
    <string>Equalization</string>
   </property>
  </action>
  <action name="actionExpansion">
   <property name="text">
    <string>Expansion</string>
   </property>
  </action>
  <action name="actionSobel">
   <property name="text">
    <string>Sobel 3*3</string>
   </property>
  </action>
  <action name="actionScharr_3_3">
   <property name="text">
    <string>Scharr 3*3</string>
   </property>
  </action>
  <action name="actionLaplacian_5_5">
   <property name="text">
    <string>Laplacian 5*5</string>
   </property>
  </action>
  <action name="actionLaplacian_7_7">
   <property name="text">
    <string>Laplacian 7*7</string>
   </property>
  </action>
  <action name="actionGaussian_3_3">
   <property name="text">
    <string>Gaussian 3*3</string>
   </property>
  </action>
  <action name="actionGaussian_5_5">
   <property name="text">
    <string>Gaussian 5*5</string>
   </property>
  </action>
  <action name="actionGaussian_7_7">
   <property name="text">
    <string>Gaussian 7*7</string>
   </property>
  </action>
  <action name="actionMean_3_3">
   <property name="text">
    <string>Mean 3*3</string>
   </property>
  </action>
  <action name="actionMean_5_5">
   <property name="text">
    <string>Mean 5*5</string>
   </property>
  </action>
  <action name="actionMean_7_7">
   <property name="text">
    <string>Mean 7*7</string>
   </property>
  </action>
  <action name="actionMedian_3_3">
   <property name="text">
    <string>Median 3*3</string>
   </property>
  </action>
  <action name="actionMedian_5_5">
   <property name="text">
    <string>Median 5*5</string>
   </property>
  </action>
  <action name="actionMedian_7_7">
   <property name="text">
    <string>Median 7*7</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
